app/styles/ember-appuniversum/_p-ember-power-select.scss

```diff
@@ -388,6 +388,7 @@ $ember-power-select-multiple-option-line-height: 1.45;
   box-shadow: $ember-power-select-dropdown-box-shadow;
   overflow: hidden;
   color: $ember-power-select-text-color;
+  z-index: 10000;
```

Comment:
Temporary z-index fix. Otherwise the dropdown is displayed below the modal backdrop.